aidial_adapter_openai/app_config.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict, List
+
+from pydantic import BaseModel, Field
+
+import aidial_adapter_openai.env as env
+
+
+class ApplicationConfig(BaseModel):
+    MODEL_ALIASES: Dict[str, str] = Field(default_factory=dict)
```

Comment:
You don't need to use `Field` with pydantic.
`MODEL_ALIASES: Dict[str, str] = {}` is enough.